examples/testing-module/Makefile

```diff
@@ -48,7 +48,7 @@ check-run-them-fedmsg-testmodule: prepare-nspawn
 	@true
 
 check-exceptions: prepare-nspawn
-	MODULEMDURL=NOT_VALID_URL  MODULE=nspawn $(CMD) --show-job-log simpleTest.py | grep 'raise ConfigExc' >/dev/null
+	MODULEMDURL=XXX  MODULE=nspawn $(CMD) simpleTest.py  | grep 'raise ConfigExc' >/dev/null
```

Comment:
It looks like this PR wasn't rebased on devel correctly. 
https://github.com/fedora-modularity/meta-test-family/commit/b227166546047eae3edebd8e3575ad0ff91d6e35#diff-db30a455353c05f6624d269ab3aa56d4R51